A/aws_c_auth/build_tarballs.jl

```diff
@@ -38,7 +38,7 @@ products = [
 dependencies = [
     Dependency("aws_c_cal_jll"; compat="0.8.3"),
     Dependency("aws_c_http_jll"; compat="0.9.3"),
-    Dependency("aws_c_sdkutils_jll"; compat="0.2.3"),
+    Dependency("aws_c_sdkutils_jll"; compat="0.2.4"),
```

Comment:
This would need to update the version number of the package.  But also, changing the compat 0.2.3 -> 0.2.4 is likely unnecessary, it's more important. when changing across breaking version (e.g. if jumping to 0.3)

Reply:
Here's my thought process:
* I'm not super worried in this case about doing a new build/release of aws_c_auth, because it hasn't actually had a new release
* But the next time it _does_ have a release, I'd ideally use the latest version of all dependencies (that's how they have the aws crt libraries set up to work in practice, each essentially relying on the latest version of deps)
* But instead of having to go out and parse all the latest versions of each dependency on each new release, I'll just update any _downstream_ dependencies when an upstream has a new release.

Does that sound like an ok plan?

Reply:
Ah, but now I see that it's still trying to build the aws_c_auth library; is there a way I can add a comment to only build a single designated library? or to skip the others?

Reply:
No, it's all or nothing